public/components/getting_started/home.tsx

```diff
@@ -3,16 +3,23 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React from 'react';
+import React, { useState, useMemo } from 'react';
 import { HashRouter, Route, RouteComponentProps, Switch } from 'react-router-dom';
-import { TraceAnalyticsCoreDeps } from '../trace_analytics/home';
 import {
   ChromeBreadcrumb,
   ChromeStart,
   HttpStart,
+  MountPoint,
   NotificationsStart,
+  SavedObjectsStart,
 } from '../../../../../src/core/public';
+import {
+  DataSourceManagementPluginSetup,
+  DataSourceSelectableConfig,
+} from '../../../../../src/plugins/data_source_management/public';
+import { TraceAnalyticsCoreDeps } from '../trace_analytics/home';
```

Comment:
Why do we have these here? `AppAnalyticsCoreDeps = TraceAnalyticsCoreDeps;`

Reply:
Removed.